### PR TITLE
Format value of HTML `class` attribute

### PR DESCRIPTION
**Pull request by fisker**
Refs: #5479 #5482

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: changelog_unreleased/html/pr-7555.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+<!-- Prettier stable -->
+<div
+  class="  foo
+bar baz"
+></div>;
+
+<!-- Prettier master -->
+<div class="foo bar baz"></div>
+```
```
Comment (evilebottnawi) [inline changelog_unreleased/html/pr-7555.md]: Can you add example with many clases when we don't fit in line?

Comment (fisker) [reply changelog_unreleased/html/pr-7555.md]: done

Comment (evilebottnawi) [reply changelog_unreleased/html/pr-7555.md]: /cc @prettier/core What do you think, maybe we should print each class on newline if we don't fix in line for readability?

Comment (lipis) [reply changelog_unreleased/html/pr-7555.md]: hmm.. that's quite an edge case.. but maybe ignore the filelength in that case?

```html
<div
  class="another element with so many classes even can not fix one line really a lot and lot of classes"
></div>
```

WDYT?

Comment (fisker) [reply changelog_unreleased/html/pr-7555.md]: @evilebottnawi I tried print each on newline , seems not good, it's common to have many classes, and we even don't print css on newline (#7556) , css normally longer then class names.

Comment (fisker) [reply changelog_unreleased/html/pr-7555.md]: @lipis I don't think this is a edge case, and not a good idea to ignore the line length

[This case](https://deploy-preview-7555--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAeAFjAtgGwHwA6UABMagEYQAmAnoScUaalQJYBuTxYOAhgM78AvARC8AtOThZJOCGADWxCVJnk5i8eIDurGOnFZqrAGZwAToxBE8zNpwY8Bw0Sumz5CgPpe4OaQgwypLu6p7ifgGwWrr6hsZmlrw4-BDivFAQ+hYeiqI2LBxcToIiYiFqGt6+-liBwaq5ChG1gTF6BkZsicopaRlZ6Dlhij6RdbD5UHioAPT29KTM8xyLZCvseCAANCAQAA4wrND8yKC85uYQ2gAKFwinKMnavDSnu+TmvIpwMADK+2+rCgAHNkDBzABXOC7TC4ADq6D0cH4gLAcD+Dz0HD0NGQ4EEOxAwP4FhgNy+IKwvGQJmSpN2ACt+AAPABCXx+-14dQAMsC4LT6TCQMyWX9gSD-ABFSFZQVIOkpEWA8yk8z4uE4In7czAmDw1hUfTIAAcAAZdrqIKT4V99vjdSiLOxBbsAI5y+AUg6PMT8cRQOBwKghonmOCe1gRim8Kk0xXC3akrCscFQkX8SUyr0KpUMkAwXjkQ3G9DIABMuwhvFYOElAGEIFhqfiUQBWImQ0kAFWLj3zIvY0IAklBQ7A-mA9YcAILjv4wGj+IXKgC+a6AA) , I think could be a real world case.

Comment (evilebottnawi) [reply changelog_unreleased/html/pr-7555.md]: I think respect line width in that cases is bad idea, maybe print it in one line always is better

Comment (evilebottnawi) [reply changelog_unreleased/html/pr-7555.md]: It is really hard to read and will create big git diff when you add a new class

Comment (fisker) [reply changelog_unreleased/html/pr-7555.md]: Actually I'm ok with both, so just `.split(/\s/).join(' ')` ?  So simple, :)

Comment (evilebottnawi) [reply changelog_unreleased/html/pr-7555.md]: Sounds good for me to inline that, will be great to hear other opinion

**Pull request merged**